src/ui/public/notify/notifier.js

```diff
@@ -97,10 +97,14 @@ function add(notif, cb) {
     });
   } else if (notif.customActions) {
     // wrap all of the custom functions in a close
-    notif.customActions = notif.customActions.map(action => {
+    notif.customActions = notif.customActions.map((action, idx) => {
       return {
         key: action.text,
-        callback: closeNotif(notif, action.callback, action.text)
+        callback: closeNotif(notif, action.callback, action.text),
+        getButtonClass() {
+          if (idx === 0) return `btn-primary btn-${notif.type}`;
+          return `btn-default btn-${notif.type}`;
+        }
```

Comment:
I edited my suggestion above to reflect the role of `index` and the classes it determines.